tools/include/eosio/codegen.hpp

```diff
@@ -272,23 +175,13 @@ namespace eosio { namespace cdt {
                   ss << tn << " arg" << i << "; ds >> arg" << i << ";\n";
                   i++;
                }
-               const auto& call_action = [&]() {
-                  ss << decl->getParent()->getQualifiedNameAsString() << "{eosio::name{r},eosio::name{c},ds}." << decl->getNameAsString() << "(";
-                  for (int i=0; i < decl->parameters().size(); i++) {
-                     ss << "arg" << i;
-                     if (i < decl->parameters().size()-1)
-                        ss << ", ";
-                  }
-                  ss << ");\n";
-               };
-               if (return_ty != "void") {
-                  ss << "const auto& result = ";
-               }
-               call_action();
-               if (return_ty != "void") {
-                  ss << "const auto& packed_result = eosio::pack(result);\n";
-                  ss << "set_action_return_value((void*)packed_result.data(), packed_result.size());\n";
+               ss << decl->getParent()->getQualifiedNameAsString() << "{eosio::name{r},eosio::name{c},ds}." << decl->getNameAsString() << "(";
```

Comment:
We shouldn't have gotten rid of `call_action` and the rest as it's used for action results.

Reply:
Good catch, when I brought the `spec_stuff` branch in, it didn't have action results and I missed adding them.